app/styles/tasks.css

```diff
@@ -2,11 +2,15 @@
   --slider-colour: #424CC1;
   --slider-background: #CFD3FF;
   --thumb-background: #A1D0FF;
+  --opaque-black: #000000b3;
+  --white: #ffffff;
+  --black: #000000;
+  --green: #008000;
 }
 
 .tasks-page {
   display: flex;
-  color: black;
+  color: #000000;
```

Comment:
Can you assign the black variable to the color property instead of directing assigning the black color?